cluster/manifests/zmon-aws-agent/deployment.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     application: "zmon-aws-agent"
     version: "v149"
 spec:
-  replicas: 1
+  replicas: {{ if ne .Alias "db" }}1{{ else }}0{{ end }}
```

Comment:
I'd create an optional config item for this to avoid hardcoding the cluster name.